### PR TITLE
Wizard walkthrough

### PR DESCRIPTION
**Pull request by BehindLoader**

### CODE REVIEW

File: docker/wizard.py [hunk 1]
```diff
@@ -0,0 +1,251 @@
...
+            '.setup-wizard-info__content-title.setup-wizard-final__box-title'
+        )
+
+        submit_btn = self.find_by_css(
+            '.rc-button.rc-button--primary.js-finish'
+        )
+        assert submit_btn
+        submit_btn.click()
+
+    def test_create_bot_account(self):
```
Comment (eugulixes) [inline docker/wizard.py]: `test_creating_bot_account`

File: docker/wizard.py [hunk 2]
```diff
@@ -0,0 +1,251 @@
...
+
+        self.addr = addr
+        self.username = username
+        self.password = password
+        self.wait = wait
+
+        self.bot_name = 'meeseeks'
+        self.bot_password = 'pass'
+
+    def _wait_for_load(self, header, selector):
```
Comment (eugulixes) [inline docker/wizard.py]: `_wait_until_loading_is_completed`

File: docker/wizard.py [hunk 3]
```diff
@@ -0,0 +1,251 @@
...
+        # Do not send welcome email
+        welcome_ckbx = self.find_by_css('label[for="sendWelcomeEmail"]')
+        assert welcome_ckbx
+        welcome_ckbx.first.click()
+
+        save_btn = self.find_by_css('.rc-button.rc-button--primary.save')
+        assert save_btn
+        save_btn.first.click()
+
+    def test_add_bot_permissions(self):
```
Comment (eugulixes) [inline docker/wizard.py]: `test_add_permissions_to_bot`

File: run_tests_in_container.sh [hunk 4]
```diff
@@ -27,5 +27,9 @@ if [[ -f .env ]]; then
     done < .env
 fi
 
-docker run -it --net=host --rm ${ARGS[@]} rocketchat-tests-based-on-splinter $*
+cd ./docker/
+
+docker-compose rm -v -f
+docker-compose up --abort-on-container-exit --force-recreate
+# docker run -it --net=host --rm ${ARGS[@]} rocketchat-tests-based-on-splinter $*
```
Comment (eugulixes) [inline run_tests_in_container.sh]: Let's get rid of the line.

File: docker/docker-compose.yml [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+      - "redis:redis"
+    environment: 
+      # Rocket.Chat
+      - AUTH_ATTEMPTS=240
+      - EXTERNAL_SCRIPTS=git:tolstoyevsky/hubot-viva-las-vegas,git:tolstoyevsky/hubot-vote-or-die,git:tolstoyevsky/hubot-pugme
+      - ROCKETCHAT_PASSSWORD=pass
+      - ROCKETCHAT_URL=http://rocketchat:8006
+      - ROCKETCHAT_USER=meeseeks
+      # Redis
+      - REDIS_URL=redis://redis:16379
```
Comment (eugulixes) [inline docker/docker-compose.yml]: I suggest we will remove `REDIS_CONF_port` and use only `REDIS_URL=redis://redis` without port specifying.

**Comment by sibiryoff** [comment]
Could you please provide some information on what this PR is created for or what issue it is going to close?

File: docker/docker-compose.yml [hunk 6]
```diff
@@ -0,0 +1,51 @@
...
+    links:
+      - "rocketchat:rocketchat"
+    environment: 
+      - USERNAME=admin
+      - PASSWORD=qwerty12+
+      - ADDR=rocketchat
+      - PORT=8006
+      - HOST=http://rocketchat:8006
+      - BOT_NAME=meeseeks
+      - BRANCH=wizard-integration
```
Comment (eugulixes) [inline docker/docker-compose.yml]: * the branch doesn't exist any more
* use `${BRANCH}` here to allow changing the branch via `.env`

**Pull request merged**